object-alloc-test/README.md

```diff
@@ -0,0 +1,8 @@
+<!-- Copyright 2017 the authors. See the 'Copyright and license' section of the README.md file at the top-level directory of this repository.
+
+Licensed under the Apache License, Version 2.0 (the LICENSE file). This file may not be copied, modified, or distributed except according to those terms. -->
+
+object-alloc-test
+=================
+
+Tests for object allocators. This crate provides mechanisms to test implementations of the `ObjectAlloc` trait for correctness.
```

Comment:
I think it's worth mentioning the corruption checker here as well

Reply:
Mmmk.